k8s/namespaces/admin/traefik2/patches/ptlsbox/cluster-00/traefik2.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   name: traefik2
   namespace: admin
 spec:
+  entryPoints:
+    websecure:
+      address: ":443"
+      forwardedHeaders:
+        insecure: true
```

Comment:
link to docs?

Reply:
https://doc.traefik.io/traefik/v2.3/routing/entrypoints/